matsim/src/main/java/org/matsim/core/mobsim/qsim/changeeventsengine/NewNetworkChangeEventsEngine.java

```diff
@@ -36,7 +36,7 @@ public void onPrepareSim() {
 	}
 	
 	private void addNetworkChangeEventToMessageQ(NetworkChangeEvent changeEvent) {
-		Message m = new Message() {
+		Message m = new Message(null) {
```

Comment:
This seems dangerous - can the event manager be passed into the method to be used?

Reply:
Right, I moved the eventManger to EventMessage. Only the descendants of EventMessage need the eventManger.